app/Helper/Helper.php

```diff
@@ -155,4 +155,20 @@ public static function PaymentCompleted($order): bool
     {
         return $order->amortization->sum('actual_amount') >= $order->amortization->sum('expected_amount');
     }
-}
+
+    public static function getEnglishOrdinalSuffix($n)
```

Comment:
Add a docstring to the function, what it does and example.

Reply:
ok

Reply:
done